packages/app/src/__deprecated__/renderer/components/core/tooltip/element-with-tooltip.component.tsx

```diff
@@ -101,6 +105,26 @@ const ElementWithTooltip: FunctionComponent<Props> = ({
     }
   }
 
+  useLayoutEffect(() => {
+    const trigger = () => {
+      if (showIfTextEllipsis) {
+        const text = ref.current as HTMLDivElement
+        const isEllipsisActive = text && text.offsetWidth < text.scrollWidth
+        setIsEllipsis(isEllipsisActive ? isEllipsisActive : false)
+      }
+    }
+
+    if (ref.current) {
+      if ("ResizeObserver" in window) {
+        new ResizeObserver(trigger).observe(ref.current)
+      }
+
+      trigger()
+    }
+  }, [ref, showIfTextEllipsis])
+
+  const showTooltip =
+    (showIfTextEllipsis && isEllipsis) || (!showIfTextEllipsis && !isEllipsis)
```

Comment:
Wouldn't it be the same (and maybe less misleading)? When `showIfTextEllipsis` is false, `isEllipsis` is never set so it is always false.

```suggestion
  const showTooltip = (showIfTextEllipsis && isEllipsis) || !showIfTextEllipsis
```

Reply:
Changed in [30aa052](https://github.com/mudita/mudita-center/pull/1030/commits/30aa0526662289c0a91e42b79d17d013c80a0236)